src/styles/blocks/header.scss

```diff
@@ -0,0 +1,63 @@
+@import "./utils/variables.scss";
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: $color-light;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  height: 60px;
```

Comment:
do you remember the requirement from the moyo-header task about height? don't set it directly to the header